search-commons/src/main/java/no/unit/nva/search/common/builder/OpensearchQueryExists.java

```diff
@@ -0,0 +1,68 @@
+package no.unit.nva.search.common.builder;
+
+import no.unit.nva.search.common.enums.ParameterKey;
+import org.opensearch.index.query.BoolQueryBuilder;
+import org.opensearch.index.query.ExistsQueryBuilder;
+import org.opensearch.index.query.QueryBuilder;
+
+import java.util.Map.Entry;
+import java.util.function.BiConsumer;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import static no.unit.nva.search.common.constant.Functions.queryToEntry;
+import static no.unit.nva.search.common.constant.Words.KEYWORD_FALSE;
+import static org.opensearch.index.query.QueryBuilders.boolQuery;
+import static org.opensearch.index.query.QueryBuilders.existsQuery;
+
+/**
+ * @author Stig Norland
+ */
+public class OpensearchQueryExists<K extends Enum<K> & ParameterKey> extends OpensearchQuery<K> {
+
+    public static final String EXISTS_ANY = "ExistsAny-";
```

Comment:
So this one will be used when API user asks if some fields with some values exist/are present on a resource not matter value?

Reply:
correct